src/components/notifications/ConfigurationTab.tsx

```diff
@@ -536,15 +536,15 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
     render() {
         if (this.state.view === ViewType.LOADING) {
             return (
-                <div style={{ height: 'calc(100vh - 172px)' }}>
+                <div className='height_reduce_to_172'>
```

Comment:
Change it to className="height_reduce_to_172"

Reply:
done